tests/test_lab_dev/test_transformations.py

```diff
@@ -274,41 +275,41 @@ def test_representation(self):
             rep1.A,
             [
                 [
-                    [-0.09966799 - 0.0j, 0.99502075 + 0.0j],
-                    [0.99502075 + 0.0j, 0.09966799 + 0.0j],
+                    [0.0 + 0.0j, 0.99500417 + 0.09983342j],
```

Comment:
Thanks Kasper, I don't know how the wrong tests managed to find their way in (@sylviemonet ?)

Reply:
@ziofil, because the test was using the one with Sgate and it is Sgate triples there as well....